pkg/hostagent/mount.go

```diff
@@ -49,34 +54,35 @@ func (a *HostAgent) setupMount(ctx context.Context, m limayaml.Mount) (*mount, e
 	if *m.SSHFS.FollowSymlinks {
 		sshfsOptions = sshfsOptions + ",follow_symlinks"
 	}
-	logrus.Infof("Mounting %q", expanded)
+	logrus.Infof("Mounting host %q to guest %q", location, mountPoint)
```

Comment:
The meaning here is clear, but if we adopt the "$LOCATION on $MOUNTPOINT" style below, then I would use it here as well:

```suggestion
	logrus.Infof("Mounting %q on %q", location, mountPoint)
```